pdfbox/pom.xml

```diff
@@ -75,6 +75,12 @@
             <artifactId>jbig2-imageio</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
They may have some other unit test package they prefer instead. Is there a "readme" for contributing to their repo? It may explain what they like to see.

Reply:
So far, I've been using this page:

https://pdfbox.apache.org/support.html

as my source of truth for contributing.  But maybe there is something more detailed in the codebase itself.  I'll check.

Reply:
Well, there's this in the README of the source code
https://github.com/apache/pdfbox/blob/trunk/README.md#contribute

That has a little more info, but nothing specific to preferred test packages.  FWIW, I don't see any previous use of mocking.  There are things that are called fooMockBar, but they are concrete objects, not a mock from a package like Mockito.

Reply:
Well, I'm sure they will gently slap you on the wrist if they need something different.

Reply:
Yeah.  If they don't want mockito, maybe I could figure out a way to do something similar with a TrueTypeFont subtype, and override on the getter and a way to set permissions masks.  That seems awkward since there is already a mature tool to do this stuff.